src/tools/__mocks__/RelayTestUtils.js

```diff
@@ -469,8 +469,14 @@ var RelayTestUtils = {
             RelayMetaRoute.get('$RelayTestUtils'),
             {}
           );
-          var actualQuery = actual.getQuery(fragment);
-          var expectedQuery = expected.getQuery(fragment);
+          const mockStore = {
+            getType() {
+              return 'RelayTestUtils';
+            },
+          };
```

Comment:
marking as WIP because this is gross and i'm hoping to think of a better option.